src/foraging_gui/Foraging.py

```diff
@@ -923,7 +958,14 @@ def _GetSettings(self):
             'newscale_serial_num_box4':'',
             'show_log_info_in_console':False,
             'default_ui':'ForagingGUI.ui',
-            'open_ephys_machine_ip_address':''
+            'open_ephys_machine_ip_address':'',
+            'metadata_dialog_folder':os.path.join(self.SettingFolder,"metadata_dialog")+'\\',
+            'rig_metadata_folder':os.path.join(self.SettingFolder,"rig_metadata")+'\\',
+            'project_infor_file':os.path.join(self.SettingFolder,"Project Name and Funding Source v2.csv"),
+            'go_cue_decibel_box1':60,
```

Comment:
What is this specifying? Where was it set before?

Reply:
`project_info_file` was from Saskia, which is required to automatically load the project information to generate project description. 

I added the `go_cue_decibel_box#` to define the go cue decibel. We measured them before, and the decibel was around 60db. The go cue decibel is currently missing from the rig metadata. When we have the decibel in the rig metadata, we should directly read from rig metadata instead of from here.

Reply:
Lets move `go_cue_decibel` to the rig metadata now

Reply:
Created issue: https://github.com/AllenNeuralDynamics/dynamic-foraging-task/issues/468